scripts/update.sh

```diff
@@ -8,7 +8,7 @@ main() {
 	export DEBIAN_FRONTEND="noninteractive"
 	if command -v apt-get 2&>/dev/null; then
 		apt-get -qq update -o Dir::Etc::SourceParts=/dev/null -o APT::Get::List-Cleanup=no -o Dir::Etc::SourceList="sources.list.d/digitalocean-agent.list"
-		apt-get -qq install -y --only-upgrade do-agent
+		apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -qq install -y --only-upgrade do-agent
```

Comment:
This tells the cron job to use the new script only if the old script is missing or unchanged

Reply:
https://manpages.debian.org/wheezy/dpkg/dpkg.1.en.html